oam/install_scripts/mcs-stop-controllernode.sh.in

```diff
@@ -1,3 +1,2 @@
 #!/bin/bash
-kill -15 $(pidof workernode)
-kill -15 $(pidof controllernode)
+systemctl stop mcs-workernode --ignore-dependencies
```

Comment:
Properly call stop on workernode from controllernode without causing a deadlock between services where both eventually timeout waiting for eachother.

kill on controllernode is not needed as systemd will run a kill -15 / check timeout / kill -9 automatically

Reply:
We must send TERM to controllernode first stop local wokernode then pool controllernode in a loop using mcsgetconfig SystemConfig DBRMTimeout as a TO. If controllernode terminates itself we quit kill it manually otherwise.